src/main/java/com/google/sps/data/FirestoreDB.java

```diff
@@ -312,6 +344,22 @@ public Query recipesMatchingTags(Iterable<String> tagIds, Iterator<String> iter)
     return DBUtils.recipeMetadata();
   }
 
+  // todo: add to interface (returns list of recipeIDs matching any of the tags)
+  public List<RecipeMetadata> recipesMatchingAnyTags(List<String> tagIds) {
+    CollectionReference recipes = DBUtils.recipeMetadata();
+    Set<RecipeMetadata> metadata = new HashSet<RecipeMetadata>();
+    for (String tagId : tagIds) {
+      if (metadata.size() >= MAX_RECIPES_PER_REQUEST) {
+        break;
+      }
+      metadata.addAll(
+          getRecipeMetadataQuery(recipes.whereEqualTo("tagIds." + tagId, true), SortingMethod.TOP));
```

Comment:
This is probably not the way we want to go about doing this... considering that there's no limit on the number of recipes in each tag, eventually, the feed will consist of literally just the one tag.

Also: it might be better to have a timestamp limit rather than a hard recipe limit, so maybe change to using that?
For example, the limit might be recipes from last week for now, and later, we can think about changing that as a potential feature.

Reply:
@bradleypickard @hari387 im not sure how hard it would be, but we could try a top of last week, top of last month, top of all time sort of thing... for now i'll hard code as Last Week since getting a lot more recipes than that is going to be significantly more complicated